Domain/PocketContainerExtensions.cs

```diff
@@ -77,5 +77,12 @@ public static PocketContainer DefaultToJsonSnapshots(this PocketContainer contai
                 return null;
             });
         }
+
+        public static PocketContainer UseJsonSerializerSettingsForSerialization(this PocketContainer container)
+        {
```

Comment:
Why not register delegates to the global settings object?

Also, any need for this to be public? I think this is just the default behavior.

Reply:
no need to be public.  I can make it internal and also add it to the global settings object.